src/_includes/partials/breadcrumbs.njk

```diff
@@ -70,7 +70,7 @@ Canada.ca > Digital Accessibility Toolkit > How to’s > Create web content > Ac
 			{% endif %}
 
 			<!-- Additional breadcrumbs based on tags and subjects -->
-			{% if tags[0] != "main" %}
+			{% if tags[0] != "main" and url != locale %}
```

Comment:
I like that, would that also cover the `/locale/index.html` URLs? If so, we could remove the `main` tag

Reply:
Just tested, `if tags[0] != "main"` is definitely still needed, otherwise, it creates non-visible/non-descript link on nested pages

Reply:
yeah I'm using it for other parts. I need to take time looking more into how I originally set up this whole project.